packages/features/ee/billing/api/webhook/__handler.ts

```diff
@@ -0,0 +1,62 @@
+/// <reference types="stripe-event-types" />
+import { buffer } from "micro";
+import type { NextApiRequest } from "next";
+import type Stripe from "stripe";
+
+import stripe from "@calcom/app-store/stripepayment/lib/server";
+import { HttpError } from "@calcom/lib/http-error";
+
+/** Stripe Webhook Handler Mappings */
+export type SWHMap = {
+  [T in Stripe.DiscriminatedEvent as T["type"]]: {
+    [K in keyof T as Exclude<K, "type">]: T[K];
+  };
+};
+
+export type LazyModule<D> = Promise<{
+  default: (data: D) => unknown | Promise<unknown>;
+}>;
+
+type SWHandlers = {
+  [K in keyof SWHMap]?: LazyModule<SWHMap[K]["data"]>;
+};
+
+/** Just a shorthand for HttpError  */
+class HttpCode extends HttpError {
```

Comment:
Style nitpick, I personally prefer
```typescript
const throwHttpCode = (statusCode: number, message: string) => throw new HttpError({ statusCode, message });
```

Reply:
Yeah. For some reason the verbose version gives me an ice